source/Spatch/scripts/search.js

```diff
@@ -2,8 +2,83 @@
 Function that toggles between showing filter options
 */
 function toggleFilter(){
-  var filterPopUp = document.getElementById('showpop');
-  filterPopUp.classList.toggle("show");
+  let filterPopUp = document.getElementById('showpop');
+  filterPopUp.classList.toggle('show');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group18/cse110-fa21-group18/pullRequest?prid=8495384)
```suggestion
  filterPopUp.classList.toggle("show");
```